release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -54,6 +54,20 @@ This release adds improvements related to the following components and concepts.
 {op-system-first} Amazon Machine Images (AMIs) are now available for AWS GovCloud regions. The availability of these AMIs improves the installation process because you are no longer required to upload a custom RHCOS AMI to deploy a cluster.
 //For more information on installing to a AWS GovCloud region, <insert link to topic after it merges>
 
+[id="ocp-4-10-conditional-updates"]
+=== Conditional updates
+
+{product-title} 4.10 adds support for consuming conditional update paths provided by the OpenShift Update Service.
+Conditional update paths convey identified risks and the conditions under which those risks apply to clusters.
+The Administrator perspective on the web console only offers recommended upgrade paths for which the cluster does not match known risks.
+However, `oc` 4.10 or later can be used to display additional upgrade paths for {product-title} 4.10 clusters.
+Associated risk information including supporting documentation references is displayed with the paths.
+The administrator may review the referenced materials and choose to perform the supported, but no longer recommended, upgrade.
```

Comment:
Is it necessary to add the last comma? 
`the supported, but no longer recommended upgrade.` better?

Reply:
@shellyyang1989 Yes, for grammatical purposes, it is necessary.

Reply:
Ack.